packages/@aws-cdk/aws-codepipeline-actions/test/integ.pipeline-s3-deploy.expected.json

```diff
@@ -311,7 +311,7 @@
                     "Ref": "DeployBucket67E2C076"
                   },
                   "Extract": "true",
-                  "CannedACL": "PublicRead",
+                  "CannedACL": "public-read",
```

Comment:
Did you actually deploy this template? Because I'm wondering how could that work, with both of those values (I'm 100% certain I deployed this template when I did the original change)...

Or does it manifest itself not when deploying the pipeline itself, but when actually executing this action?

Reply:
I did deploy this. 
So, Access-control on a bucket only accepts camel case (as documented).
But, CannenACL on this action accepts both the cases (for me, but not for everyone), even though the docs say that it should only accept kebab-case.

Reply:
I did an experiment myself, and you are correct sir! With CamelCase, the action fails with:

```
Invalid action configuration
Invalid canned ACL: BucketOwnerFullControl. For a list of valid options, see https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl
```

<img width="596" alt="s3-deploy-pipeline-error" src="https://user-images.githubusercontent.com/460937/73229159-a40ef980-412d-11ea-9fe0-ab5316bcf91b.png">